src/components/slider/slider.spec.ts

```diff
@@ -588,6 +590,65 @@ describe('MdSlider', () => {
       expect(sliderContainerElement.classList).toContain('md-slider-active');
     });
   });
+
+  describe('slider with two-way binding', () => {
```

Comment:
You've got a one-way binding in this test :)   Pet peeve, but can you change to read "slider as a custom form control"?